src/postamble.js

```diff
@@ -125,6 +125,10 @@ Module['then'] = function(func) {
 };
 #endif
 
+/**
```

Comment:
Ok, when `WASM=1` we don't run the js optimizer, yeah. In that case you can remove the `--js-opts 2`, as it does nothing.

Why do you invoke closure manually instead of passing `--closure`?